### PR TITLE
REVIEW: XMPP Newsletter August 2022

### PR DESCRIPTION
**Pull request by Echolon**
## This is the draft for the next release covering August 2022.

Final release is planned for 5th of September 2022.

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: [commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)

[You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links pray

Just paste your prepared text in this thread below or in our XMPP channel (xmpp:[commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)?join)

Please do not just drop bare links - rather try to make a short and decent formulated description (<5 lines) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on!

XMPP Newsletter Timeline

Status:

* [x] Collection of news
* [x] Review
* [ ] Blog post + Tiny Newsletter
* [ ] Social media
* [ ] Translations
* [ ] Archived ([Use pandoc!](https://pandoc.org/))

### CODE REVIEW

File: content/blog/2022-09-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+## Articles
+
+_Please do not simply drop links, rather try to make a short (<5 lines) and decent formulated description of what is the main outcome of your source. This will help us a lot! You do not need to use a hard, formal tone, be passionate about why this motivates you or what you work on!_
+
+## Software news
+
+### Clients and applications
+
+_Please do not simply drop links, rather try to make a short (<5 lines) and decent formulated description of what is the main outcome of your source. This will help us a lot! You do not need to use a hard, formal tone, be passionate about why this motivates you or what you work on!_
+
```
Comment (melvo) [inline content/blog/2022-09-05-newsletter.md:58]:
```suggestion

Kaidan, the XMPP client for every device, released a [post about its end-to-end encryption trust management](https://www.kaidan.im/2022/08/31/e2ee-trust-management/)

```

Comment (licaon-kter) [reply content/blog/2022-09-05-newsletter.md:58]: @melvo add it to https://yopad.eu/p/xmpp-newsletter-365days so it gets pulled when that is

Comment (melvo) [reply content/blog/2022-09-05-newsletter.md:58]: @licaon-kter Thanks for the hint. @Echolon told me that comments are also welcome but I just saw that on https://yopad.eu/p/xmpp-newsletter-365days that there already was an entry for Kaidan's post. I updated that entry. Thus, my suggested change here is obsolete.

Comment (Echolon) [reply content/blog/2022-09-05-newsletter.md:58]: @melvo actually I would prefer a PR in this case AND one that comes with one of your images?

**Comment by Echolon** [comment]
@wurstsalat3000 can you check rendering already and the xep section if I copied correctly. I have a bad feeling

File: content/blog/2022-09-05-newsletter.md (hunk 1 above)
Comment (melvo) [reply content/blog/2022-09-05-newsletter.md:84]: @Echolon I created #1167 for that.

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter August 2022 release

- Many thanks to everyone!!! This release to **Adrien Bourmault (neox), anubis, Anoxinon e.V., Benoît Sibaud, cpm, daimonduff, emus, Gooya, Holger, *IM*, Ludovic Bocquet, martin, MattJ, MSavoritias (fae,ve), nicfab, Pierre Jarillon, Sam Whited, TheCoffeMaker, wh0nix, vanitasvitae, wurstsalat, Zash** for their help in creation, review and translation!

- :heart: :fireworks:  Next release is next month on **5th of October 2022**!

Have a good time until then and see you soon!

**Pull request merged**